playground/backend/internal/utils/precompiled_objects_utils.go

```diff
@@ -38,3 +40,54 @@ func PutPrecompiledObjectsToCategory(categoryName string, precompiledObjects *cl
 	}
 	sdkCategory.Categories = append(sdkCategory.Categories, &category)
 }
+
+// GetCatalogFromStorage returns the precompiled objects catalog from the cloud storage
+func GetCatalogFromStorage(ctx context.Context) ([]*pb.Categories, error) {
```

Comment:
Is it possible to add tests for this method?

Reply:
No, its logic is connected with retrieving the data from the bucket – testing this will be integration testing, which we don't do now.